### PR TITLE
Placement: Add Victoria Placement Service

### PR DESCRIPTION
**Pull request by jsaroja66**
# PR description

Provide support for the new Placement service APIs introduced in Openstack Victoria release.
OpenStack API documentation:
https://docs.openstack.org/api-ref/placement/
Changes ware verified against an OpenStack Victoria instance.

...

## Submitter checklist

Make sure that following is addressed to make the PR easier to process:

- [ ] If applicable, changes are covered by either a unit or a functional test.
- [x] If applicable, changes ware verified manually against an OpenStack instance.
- [x] If the change is API related, the PR description links to OpenStack API documentation of that particular endpoint/request (https://docs.openstack.org/api-quick-start/#current-api-versions).
- [x] If the change concerns particular OpenStack service, its name is used as a PR name prefix (ex.: "Neutron: Add floatingIp port forwardings service").
- [ ] If the PR closes existing GitHub issue, PR name have prefix: `Fix #NNN: `.

### CODE REVIEW

File: core/src/main/java/org/openstack4j/api/Apis.java [hunk 1]
```diff
@@ -124,7 +136,6 @@ public static ImageService getImageService() {
 
     /**
      * Gets the (Glance) Image v2 services API
-     *
```
Comment (olivergondza) [inline core/src/main/java/org/openstack4j/api/Apis.java:127]: Please avoid modifying unrelated lines.

File: core/src/main/java/org/openstack4j/api/OSClient.java [hunk 2]
```diff
@@ -278,45 +286,10 @@
...
-     *
-     * @return the DNS service
-     */
-    DNSService dns();
-
     /**
      * OpenStack4j Client which authenticates against version V2
      */
-    interface OSClientV2 extends OSClient<OSClient.OSClientV2> {
+    public interface OSClientV2 extends OSClient<OSClient.OSClientV2> {
```
Comment (olivergondza) [inline core/src/main/java/org/openstack4j/api/OSClient.java:292]: Unrelated and unneeded. All declarations in interfaces are public by default.

**Review by olivergondza — CHANGES_REQUESTED** [review]
Thank you for your Pull Request. Please revert all changes unrelated to your main work, so this can be reviewed. Reindenting and reordering just hides your true contribution.

File: core/src/main/java/org/openstack4j/api/OSClient.java (hunk 2 above)
Comment (jsaroja66) [reply core/src/main/java/org/openstack4j/api/OSClient.java]: I have reverted all indentation changes and updated the commit. Please review.
